src/garage/replay_buffer/her_replay_buffer.py

```diff
@@ -11,23 +10,33 @@
 
 
 def make_her_sample(replay_k, reward_fun):
-    """
-    Generate a transition sampler for HER ReplayBuffer.
+    """Generate a transition sampler for HER ReplayBuffer.
+
+    Args:
+        replay_k (float): Ratio between HER replays and regular replays
+        reward_fun (callable): Function to re-compute the reward with
+            substituted goals
+
+    Returns:
+        callable: A function that returns sample transitions for HER.
 
-    :param replay_k: the ratio between HER replays and regular replays
-    :param reward_fun: function to re-compute the reward with substituted goals
-    :return:
     """
     future_p = 1 - (1. / (1 + replay_k))
 
     def _her_sample_transitions(episode_batch, sample_batch_size):
-        """
-        Generate a dictionary of transitions.
+        """Generate a dictionary of transitions.
+
+        Args:
+            episode_batch (dict): Original transitions which
+                transitions[key] has shape [batch_size, T, dim]
```

Comment:
see #1058 for a style for documenting argument shapes. maybe we should update `CONTRIBUTING.md` with it.

Reply:
Okay, I updated `CONTRIBUTING.md` so please use that style.